### PR TITLE
Fix german translation of US zone names

### LINKED ISSUE #3811: Review odd German zone-names of US-zones
Hi all,
the translation of most of the US-zone names is very odd in German, seems like it was translated automatically? They should remain being named in English.

my favourites:
grafik

grafik

grafik

### PR DESCRIPTION
**Pull request by DerTimonius**
Thanks for this great website, stumbled over it just today! 🙌 

## Issue

Closes #3811

## Description

This PR will fix the weird US zone names in the german locale. More often than not I left the original name, since it's a company name and only left translations if they made sense (and changed them). 
I'm also a german native speaker, so the translations should be solid 🙂 

### Double check

Since this PR only changes the locales file, I did not run the tests.

### CODE REVIEW

File: web/public/locales/de.json [hunk 1]
```diff
@@ -1644,135 +1644,135 @@
...
     },
     "US-SW-PNM": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "New Mexiko Firma für öffentlichen Dinest"
+      "zoneName": "Public Service Company Of New Mexico"
     },
     "US-SW-SRP": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "Salzfluss Projekt"
+      "zoneName": "Salt River Projekt"
```
Comment (jarek) [inline web/public/locales/de.json:1755]: German K left over here

File: web/public/locales/de.json [hunk 2]
```diff
@@ -1644,135 +1644,135 @@
...
     },
     "US-NW-DOPD": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "PUD Nr.1 von Douglas County"
+      "zoneName": "Public Utility District Nr.1 von Douglas County"
     },
     "US-NW-GCPD": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "PUD Nr.2 von Grant County, Washington"
+      "zoneName": "Public Utility District Nr.2 von Grant County, Washington"
```
Comment (VIKTORVAV99) [inline web/public/locales/de.json:1655]: These still have "von" in them, should they?

File: web/public/locales/de.json [hunk 3]
```diff
@@ -1644,135 +1644,135 @@
...
     },
     "US-NW-PGE": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "Portland Allgemeine Elektrik Firma"
+      "zoneName": "Portland General Electric Company"
     },
     "US-NW-PSCO": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "Öffentliche Servicefirma von Colorado"
+      "zoneName": "Public Service Company Of Colorado"
```
Comment (VIKTORVAV99) [inline web/public/locales/de.json:1691]:
```suggestion
      "zoneName": "Public Service Company of Colorado"
```

File: web/public/locales/de.json [hunk 4]
```diff
@@ -1644,135 +1644,135 @@
...
     },
     "US-SW-HGMA": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "New Harquahala Erzeugungsunternehmen, LLC"
+      "zoneName": "New Harquahala Generating Company, LLC"
     },
     "US-SW-PNM": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "New Mexiko Firma für öffentlichen Dinest"
+      "zoneName": "Public Service Company Of New Mexico"
```
Comment (VIKTORVAV99) [inline web/public/locales/de.json:1751]:
```suggestion
      "zoneName": "Public Service Company of New Mexico"
```

File: web/public/locales/de.json [hunk 5]
```diff
@@ -1644,135 +1644,135 @@
...
     },
     "US-TEN-TVA": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "Tennessee Valley Behörde"
+      "zoneName": "Tennessee Valley Authority"
     },
     "US-TEX-ERCO": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "Rat für elektrische Zuverlässigkeit von Texas, Inc."
+      "zoneName": "Electric Reliability Council Of Texas, Inc."
```
Comment (VIKTORVAV99) [inline web/public/locales/de.json:1771]:
```suggestion
      "zoneName": "Electric Reliability Council of Texas, Inc."
```

File: web/public/locales/de.json [hunk 6]
```diff
@@ -1524,87 +1524,87 @@
...
     },
     "US-CAL-IID": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "Kaiserlicher Bewässerungsbezirk"
+      "zoneName": "Imperial Irrigation District"
     },
     "US-CAL-LDWP": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "Los Angeles Abteilung für Wasser und Energie"
+      "zoneName": "Los Angeles Department Of Water And Power"
```
Comment (VIKTORVAV99) [inline web/public/locales/de.json:1539]:
```suggestion
      "zoneName": "Los Angeles Department of Water and Power"
```

File: web/public/locales/de.json [hunk 7]
```diff
@@ -1620,7 +1620,7 @@
     },
     "US-MIDW-LGEE": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "Louisville Gas & Electric Company und Kentucky Utilities"
+      "zoneName": "Louisville Gas And Electric Company And Kentucky Utilities"
```
Comment (VIKTORVAV99) [inline web/public/locales/de.json:1623]:
```suggestion
      "zoneName": "Louisville Gas and Electric Company and Kentucky Utilities"
```

**Review by VIKTORVAV99 — COMMENTED** [review]
Just some grammar corrections (I know these are precent in the English version as well).

File: web/public/locales/de.json (hunk 2 above)
Comment (DerTimonius) [reply web/public/locales/de.json:1655]: I left those in on purpose. My thinking was that not many people know what a `Public Utility District` is, so I thought, *This is the PUD Nr. 2 of Grant County etc.* (in german *Das ist das Public Utility Disctrict Nummer 2 von Grant County*) makes more sense

**Pull request merged**